buildkite/download-artifact/README.md

```diff
@@ -0,0 +1,37 @@
+# buildkite/download-artifact
+
+GitHub Action to download the artifacts for a given Buildkite build.
+
+## Inputs
+
+Following inputs can be used as `step.with` keys
+
+| Name            | Type    | Default        | Description                                                            |
+|-----------------|---------|----------------|------------------------------------------------------------------------|
+| `build`         | String  |                | The Buildkite pipeline build.                                          |
+| `org`           | String  | `elastic`      | The Buildkite org.                                                     |
+| `path`          | String  |                | A file, directory or wildcard pattern that describes what to download  |
+| `pipeline`      | String  |                | The Buildkite pipeline to interact with.                               |
+| `token`         | String  |                | The Buildkite API Token.                                               |
+
+## Usage
+
+```yaml
+jobs:
+  run-buildkite:
+    runs-on: ubuntu-latest
+    steps:
+      - id: buildkite-run
+        uses: elastic/oblt-actions/buildkite/run@v1
+        with:
+          token: ${{ secrets.BUILDKITE_TOKEN }}
+          pipeline: "my-super-pipeline"
+          wait-for: true
+
+      - uses: elastic/oblt-actions/buildkite/download-artifact@v1
+        with:
+          build: ${{ steps.buildkite-run.outputs.build }}
+          path: "artifacts.tar"
+          pipeline: "my-super-pipeline"
```

Comment:
should we add the output `pipeline` to the `buildkite/run` action?

so we can do 

```
pipeline: ${{ steps.build-run.outputs.pipeline }}
```

and only need to specify it once.

In case the pipeline name is changed in the future.. you only need to do it in a single place.

Reply:
I like that

Reply:
see [`697b0cf` (#18)](https://github.com/elastic/oblt-actions/pull/18/commits/697b0cf341fd4098ab5cab139d40db3cd6071aec)